app/uk/gov/hmrc/cdsimportsddsfrontend/services/xml/DeclarationXml.scala

```diff
@@ -72,7 +74,7 @@ class DeclarationXml {
         {additionalInformation(dec.documentationAndReferences.headerAdditionalInformation)}
         {dec.parties.authorisationHolders.map(_.toXml)}
         {dec.borderTransportMeans.toXml}
-        {maybeCurrencyExchange(dec)}
+        {dec.currencyExchange.map(_.toXml).getOrElse(NodeSeq.Empty)}
```

Comment:
`dec.currencyExchange.toXml` should work?

Reply:
Done